src/map/ai/controllers/trust_controller.cpp

```diff
@@ -62,7 +62,7 @@ void CTrustController::Tick(time_point tick)
 {
     m_Tick = tick;
 
-    if (POwner->PMaster == nullptr)
+    if (!POwner->PMaster)
```

Comment:
Is this change just a matter of style more than implementation?

Reply:
Just style